composer.json

```diff
@@ -52,9 +54,28 @@
 		"cs:fix": "php-cs-fixer fix",
 		"lint": "find . -name \\*.php -not -path './vendor/*' -exec php -l \"{}\" \\;",
 		"phan": "phan --allow-polyfill-parser -k .phan/config.php",
+		"post-install-cmd": [
+			"\"vendor/bin/mozart\" compose",
```

Comment:
./vendor/bin/mozart should work without the quotes.

Reply:
That is a 1:1 copy of https://github.com/coenjacobs/mozart#scripts